### ISSUE TITLE
Texarea WCAG label reference error

### ISSUE THREAD

**Issue by jeffreiffers** [digdir/designsystem#678]
### Description of the bug

Texarea has an invalid label reference.

### Steps To Reproduce

1. Install wave accessibility extension
2. Run page (with Textarea component) test with wave

**Comment by mimarz** [comment]
Thanks for reporting this! 

We'll have a look at this when we get to do improvements on TextArea in #323

**Comment by mimarz** [comment]
This should be fixed now.
